src/data/markdown/docs/03 cloud/09 Optional Add-ons/01 Private load zones.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Private load zones'
```

Comment:
Until we decide how to restructure Cloud Docs, I would move this article to the Cloud FAQ section.

Reply:
Also, please create a link to Private Load Zones from https://k6.io/docs/cloud/#cloud-features

Reply:
Done on both, thanks!